notifications_utils/field.py

```diff
@@ -167,10 +167,13 @@ def placeholders(self):
 
     @property
     def placeholders_meta(self):
-        meta = {
-            Placeholder(body).name: {"is_conditional": Placeholder(body).is_conditional()}
-            for body in re.findall(self.placeholder_pattern, self.content)
-        }
+        meta = {}
+        for body in re.findall(self.placeholder_pattern, self.content):
```

Comment:
Looks good. A summary comment to explain why these two conditions are necessary would be nice on our selves.

Reply:
👍